apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/commons/OpenAPIContentProvider.java

```diff
@@ -0,0 +1,249 @@
+          + " represents the logical identity for a resource, or as much of that"
+          + " identity that is known. In FHIR, every resource must have a <i>logical ID<i/> which is"
+          + " defined by the FHIR specification as:"
+          + "<p><code>"
+          + "Any combination of upper or lower case ASCII letters ('A'..'Z', and 'a'..'z', numerals ('0'..'9'),"
+          + " '-' and '.', with a length limit of 64 characters. (This might be an integer, an un-prefixed OID, UUID"
+          + " or any other identifier pattern that meets these constraints.)</code></p><p>"
+          + "This class contains that logical ID, and can optionally also contain a relative or absolute URL"
+          + " representing the resource identity; the following are all valid values for IdType, and all might"
+          + " represent the same resource:</p><ul>"
+          + "<li><code>123</code> (just a resource\u0027s ID)</li>"
+          + "<li><code>beneficiary/123</code> (a relative identity)</li>"
+          + "<li><code>http://example.com/beneficiary/123 (an absolute identity)</code></li>";
+
+  /** Open API short description for /ExplanationOfBenefit's EOB claim type parameter. */
+  public static final String EOB_CLAIM_TYPE_SHORT = "A list of BFD claim types to include";
+  /** Open API content value for /ExplanationOfBenefit's EOB claim type parameter. */
+  public static final String EOB_CLAIM_TYPE_VALUE =
+      "A list of claim types that filters the response to contain only EoBs which have any of the specified claim types. "
+          + "This is optional and defaults to * (all types);"
+          + "<p/>Supported Claim Type values:<ul>"
+          + "<li>carrier</li>"
+          + "<li>inpatient</li>"
+          + "<li>outpatient</li>"
+          + "<li>hha</li>"
+          + "<li>hospice</li>"
+          + "<li>snf</li>"
+          + "<li>dme</li>"
+          + "<li>partd</li></ul><p>"
+          + "<p/>Examples:<ul>"
+          + "<li>type=carrier,inpatient,snf,dme</li>"
+          + "<li>type=outpatient</li>"
+          + "<li>type=&#42;</li>";
+
+  /** Open API short description for /ExplanationOfBenefit's serviceDate parameter. */
+  public static final String EOB_SERVICE_DATE_SHORT =
+      "Include claims whose <i>thru date</i> fall within the given range";
+  /** Open API content value for /ExplanationOfBenefit's serviceDate parameter. */
+  public static final String EOB_SERVICE_DATE_VALUE =
+      "Only satisfy the Search request if a claim\u0027s <i>thru</i> Date"
+          + " falls within a specified <i>DateRange<i/>. A DateRange can include both"
+          + " lo and hi date values, only a lo date value, or only a hi date value."
+          + "<p/>Examples:<ul>"
+          + "<li>&service-date=gt2023-01-02&service-date=lt2023-05-01</li>"
+          + "<li>&service-date=gt2023-01-02</li>"
+          + "<li>&_service-date=lt2023-05-01</li></ul>";
```

Comment:
Can we fix this to "through" "high" and "low"? Bothers me grammatically.
A slightl rewrite might be useful to note to end users that this is used to filter by the "billable period" for claims, which is the language that partners know this field as (through date is probably less useful, as I believe that's our/CCW's internal term for the field)

Reply:
Rewriting